include/matx/transforms/cub.h

```diff
@@ -878,6 +938,42 @@ inline void ExecSort(OutputTensor &a_out,
 #endif
   }
 
+  /**
+   * Execute a max on a tensor
+   *
+   * @note Views being passed must be in row-major order
+   *
+   * @tparam OutputTensor
+   *   Type of output tensor
+   * @tparam InputOperator
+   *   Type of input tensor
+   * @param a_out
+   *   Output tensor
+   * @param a
+   *   Input tensor
+   * @param stream
+   *   CUDA stream
+   *
+   */
+  inline void ExecArgMinMax(const InputOperator &in,
+                            const cudaStream_t stream)
+  {
+#ifdef __CUDACC__
+    MATX_NVTX_START("", matx::MATX_NVTX_LOG_INTERNAL)
+  
+    // Set up CUB stuff
+    auto d_in = thrust::counting_iterator<size_t>(0);
+    auto min_op = minmaxer{in.Data()};
```

Comment:
Passing the .Data() pointer will cause this to fail if the underlying tensor has been permuted.  Ideally we'd rework RandomOperatorIterator and RandomOperatorOutputIterator to work correctly with Thrust.

At a minimum, suggest updating the docs (limitations.rst) to indicate this doesn't support permutations.